src/applications/check-in/utils/validateVeteran/index.js

```diff
@@ -98,16 +103,25 @@ const validateLogin = async (
       goToErrorPage();
     } else {
       setSession(token, resp.permissions);
-      resetAttempts(window, token, true);
+      if (!isLorotaDeletionEnabled) {
+        resetAttempts(window, token, true);
+      }
       goToNextPage();
     }
   } catch (e) {
     setIsLoading(false);
     if (e?.errors[0]?.status !== '401' || isMaxValidateAttempts) {
-      goToErrorPage();
+      let params = '';
+      if (e?.errors[0]?.status === '410') {
```

Comment:
Should the `|| isMaxValidateAttempts` from 113 be on this line instead to make sure the param is triggered regardless of the feature flag.

Reply:
oh yeah, I think the I could just move the params to a const with the string, because every route to error from here would be for validation.

Reply:
actually that probably isn't 100% true. I'll go with your suggestion because it could be an non-validation type error.